client/src/util/stateManager/universe.js

```diff
@@ -98,13 +99,31 @@ function AnnotationsFBSToDataframe(arrayBuffer) {
 
 function LayoutFBSToDataframe(arrayBuffer) {
   const fbs = decodeMatrixFBS(arrayBuffer, true);
-  if (fbs.columns.length !== 2 || !fbs.columns.every(isFpTypedArray)) {
+  if (fbs.columns.length < 2 || !fbs.columns.every(isFpTypedArray)) {
     // We have strong assumptions about the shape & type of layout data.
     throw new Error("Unexpected layout data type returned from server");
   }
+
+  /*
+  TODO: XXX
+
+  Temporary code to support the progressive introduction of multi-layout
+  support.  For now, we search for one of the following in the layouts
+  and use it if we find it.  If not, we just use the first available.
+
```

Comment:
sorry, I am unclear which is extra - can you clarify?

(also keep in mind that this comment will be deleted once I finish the PR - this is only in the code because we are trying to keep the PRs small)

Reply:
line 113 is extra whitespace it seems. not a major issue.

Reply:
removed.